### ISSUE TITLE
Make development beginner-friendly

### ISSUE THREAD

**Issue by kytrinyx** [exercism/exercism.io#1507]
How can we make it easier to contribute?

TL;DR
- [ ] Create a script `bin/bootstrap`
- [ ] Describe project structure (perhaps in the `CONTRIBUTING.md` guide)
- [ ] Use the `beginner-friendly` tag
- [ ] Work through the issues list and clarify. Is there some information that would make it easier to solve?

---

Right now it there are a number of obstacles to contributing, especially if you are not comfortable with open source, or not comfortable with Ruby or Sinatra.

It's fairly onerous to get the project running locally. We should have a script that installs all the dependencies, copies files, creates and migrates databases, etc. This was discussed in #1478. There should also be good seed data that makes it easy to use app as very different users. I'm working on that in [exercism/seeds](https://github.com/exercism/seeds/issues).

There's no real rhyme or reason for what objects exist where, or why. There should be some documentation that gives the 30,000 foot overview of the codebase.

If you're new to Ruby or open source, and you look at the existing issues, it's not easy to know if that issue is a 5-month death-march, or a 3-minute tweak that nobody has gotten to yet. The issues should be fleshed out to indicate more about the size, and if it's something that a beginner could tackle, there should be some indication.

Eventually, when all of these things are done, we could add exercism to [openhatch](https://openhatch.org/guide/).
